centreon/www/install/createTablesCentstorage.sql

```diff
@@ -330,6 +330,8 @@ CREATE TABLE `resources` (
   `output` text DEFAULT NULL,
   `enabled` tinyint(1) NOT NULL DEFAULT 1 COMMENT '0=resource disabled, 1=resource enabled',
   `icon_id` bigint(20) unsigned DEFAULT NULL,
+  `flapping` tinyint(1) NOT NULL DEFAULT 0 COMMENT '0=false, 1=true',
+  `percent_state_change` float DEFAULT NULL,
```

Comment:
can you add a comment for 'percent_state_change' too ?

Reply:
will be handled in the PR for View by Host